src/commands/info/discordstatus.ts

```diff
@@ -1,15 +1,142 @@
 /* eslint-disable camelcase */
-import { Message, MessageEmbed, EmbedField } from 'discord.js';
+// import { Message, MessageEmbed, EmbedField } from 'discord.js';
```

Comment:
Confused why all of this is commented out in the first place???

Reply:
Fixed. Sometimes I'll comment out the entire command and just work under it—did not realize I left it in, sorry!